### ISSUE TITLE
Touchpad sensitivity 

### ISSUE THREAD

**Issue by devolkpl** [simov/simplr-smoothscroll#21]
Did someone tried to use this plugin with macbook's touchpad? Even the smalles interaction with touchpad gets me to the top/bottom of the page.
I guess simplr-smoothscroll wasn't tested with touch events at all.

**Comment by simov** [comment]
Yep, it wasn't tested, but you can play around with the `step` option. Set it to something smaller than the default `55`px.
